pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,42 @@
+url: https://cmap-repos.github.io/cmapplot
+
+home:
+  title: CMAP Themes and Color Palettes
+  description: >
+    Provides themes and color scales for 'ggplot2', based on Chicago
+    Metropolitan Agency for Planning (CMAP) design guidelines.
+
+authors:
+  Chicago Metropolitan Agency for Planning:
+    href: https://www.cmap.illinois.gov
+
+template:
+  params:
+    bootswatch: flatly
+
+reference:
+- title: Theme and Plotting Functions
+- contents:
+  - theme_cmap
+  - finalize_plot
+  - starts_with("geom_")
+  - integer_breaks
+- title: Color Palettes and Gradients
+- contents:
+  - cmap_gradients
+  - cmap_palettes
+  - starts_with("cmap_fill_")
+  - starts_with("cmap_color_")
+  - starts_with("cmap_colour_")
```

Comment:
Curious how @gritzenthaler and others feel about dropping the British spelling of these functions? I'm mostly indifferent, but just wondering about clutter.

Reply:
I vote to keep it for full consistency with `ggplot` (which uses British English by default), even though it's unlikely any non-CMAPers will ever use this package.

Reply:
Yeah, I was adding them because I saw similar practice in other ggplot packages. I don't think it hurts to keep them, and the code to define them is just one line per function.